### ISSUE TITLE
Undervolt turned on while on experimental UV

### ISSUE THREAD

**Issue by vylestar** [KyleGospo/Steam-Deck-Software-Undervolt#3]
I installed the latest version and ran experimental. I checked statusadj.txt right away and it confirmed experimental was on.

From here my order of operations maybe mixed up but at one point I restarted the deck and started running mprime. During the stress test, I went to check status again and it said undervolt on. continued to run the test for 30 mins with no hiccups. I returned to game mode to go turn off the undervolt and I restarted the deck. Checked the status to make sure it was off, which it was. Turned on experimental and checked status and it shows experimental.  Now I'm currently running a stress test for experimental UV.

TLDR; I'm not sure what happened, but when I ran experimental at some point undervolt was turned on. The only 2 scripts I've used so far is experimental to test stability and off (because for some reason it turned undervolt on). Never used the on.sh script (confirmed this in steam recent games, it only shows experimental.sh and off.sh)
